src/scenario.jl

```diff
@@ -626,8 +638,13 @@ function run_model(domain::Domain, param_set::NamedDimsArray, corals::DataFrame,
         # Add recruits to current cover
         C_t[p.small, :] .+= recruitment
 
-        in_shade_years = (shade_start_year <= tstep) && (tstep <= (shade_start_year + shade_years - 1))
-        in_seed_years = (seed_start_year <= tstep) && (tstep <= (seed_start_year + seed_years - 1))
+        # Years at which seeding/fogging/shading occurs
+        in_fog_years =
+            (fog_start_year <= tstep) && (tstep <= (fog_start_year + fog_years - 1))
+        in_shade_years =
+            (shade_start_year <= tstep) && (tstep <= (shade_start_year + shade_years - 1))
+        in_seed_years =
+            (seed_start_year <= tstep) && (tstep <= (seed_start_year + seed_years - 1))
```

Comment:
While you're at these changes, could you check if this works?

From memory there was something preventing me from implementing it this way

```suggestion
        in_fog_years =
            fog_start_year <= tstep <= (fog_start_year + fog_years - 1)
        in_shade_years =
            shade_start_year <= tstep <= (shade_start_year + shade_years - 1)
        in_seed_years =
            seed_start_year <= tstep <= (seed_start_year + seed_years - 1)
```

Reply:
Yeah I think `in_*_timeframe` stands out more from the `*_decision_years` parameters and is more informative

I'll check what you've suggested